keras_retinanet/models/mobilenet.py

```diff
@@ -0,0 +1,56 @@
+import keras
+from keras import backend as K
```

Comment:
Considering we use `keras.backend` everywhere, I prefer that here too. Actually that means this import can be removed.